.travis.yml

```diff
@@ -14,12 +14,5 @@ matrix:
 install:
     - source ci/install.sh
 script:  bash ci/test_script.sh
-after_success:
-  - coveralls
-cache:
-  directories:
-    - $HOME/nest-2.10.0
-    - $HOME/nrn-7.4
-    - $HOME/build/nest-2.10.0
-    - $HOME/build/nrn-7.4
-    - $HOME/.cache/pip
+#after_success:
```

Comment:
Ok, I deleted those lines since I was having issues with tests passing due to the cache (now fixed) and interaction with coveralls. I'll see what I can do to get the coveralls check working with my fork